modules/gdscript/gdscript_editor.cpp

```diff
@@ -3456,6 +3456,30 @@ ::Error GDScriptLanguage::lookup_code(const String &p_code, const String &p_symb
 				return OK;
 			}
 		} break;
+		case GDScriptParser::COMPLETION_TYPE_ATTRIBUTE: {
+			const GDScriptParser::TypeNode *type = static_cast<const GDScriptParser::TypeNode *>(context.node);
+
+			const GDScriptParser::IdentifierNode *prev;
```

Comment:
```suggestion
			const GDScriptParser::IdentifierNode *prev = nullptr;
```

Should be initialized to pass the CI tests.